core/services/ocrcommon/addressparser.go

```diff
@@ -0,0 +1,34 @@
+package ocrcommon
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
```

Comment:
nit/ Would it make sense to isolate evm stuff in a sub-package?

Reply:
oh yeah actually this is better housed just in the relay/evm package